docs/example_yamls/exoplanetary/planets1.yaml

```diff
@@ -0,0 +1,19 @@
+!PlanetarySystem
+position:
+  !Coord
+  ra: 2 deg
+  dec: 1 deg
+  distance: 100 pc
```

Comment:
Why require the distance?

I think it only makes sense to either have a relative magnitude, or an absolute magnitude and a distance, but not a distance and a relative magnitude.

Reply:
> This isn't about the magnitude. 

> This also makes it somewhat easier to do something like "Let's try pushing this system further away and see if we could still resolve it."

It does lead to confusion though, because if you push the system further away, without changing the apparent magnitude, it essentially becomes a different system, because its intrinsic magnitude would change.

Also, if distance is supported, who is responsible for any redshift of the spectrum? The user, ScopeSim_Targets, or ScopeSim? (Well, these are stars, so nearby enough that it doesn't matter, but we should prevent things from getting complicated later.)